thirdparty/runtime_environments_test.go

```diff
@@ -207,3 +207,28 @@ func TestGetHistory(t *testing.T) {
 	assert.NotEmpty(t, result)
 	assert.Len(result, 15)
 }
+
+func TestGetEvents(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	assert := assert.New(t)
+	config := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, config, "TestGetEvents")
+	c := NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+
+	// Verify that getEvents errors when not provided the required distro field.
+	_, err := c.getEvents(ctx, EventHistoryOptions{})
+	assert.Error(err)
+
+	// Verify that getEvents functions correctly with page and limit.
```

Comment:
A nice additional check might be to make sure that the timestamps are in chronological order